src/style.css

```diff
@@ -0,0 +1,130 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
```

Comment:
You shoud make separate css files for each block

Reply:
done +